hardware_interface/CMakeLists.txt

```
@@ -101,6 +104,10 @@ if(BUILD_TESTING)
   ament_add_gmock(test_component_parser test/test_component_parser.cpp)
   target_link_libraries(test_component_parser hardware_interface ros2_control_test_assets::ros2_control_test_assets)
 
+  # Test XML Schema Validator
+  ament_add_gmock(test_validate_xml_schema test/test_validate_xml_schema.cpp)
+  target_link_libraries(test_validate_xml_schema PkgConfig::LIBXML2 ament_index_cpp::ament_index_cpp)
```

see above, +reformatting, +sort ascending
```suggestion
  target_link_libraries(test_validate_xml_schema
     ament_index_cpp::ament_index_cpp
     LibXml2::LibXml2
  )
```